roottest/root/io/TFile/.rootrc

```diff
@@ -1 +0,0 @@
-Rint.History:            .root_hist
```

Comment:
What is the advantage of removing the `.rootrc`?

Reply:
It was there to avoid putting roottest invocations in global ROOT history.
But now it managed centrally and not need to be done individually in each subdirectory.

Reply:
What do you mean by 'managed' centrally?

Reply:
There is `.rootrc` already in `roottest/root` directory - actually no idea why.

As I can see all `gROOT->ProcessLine` invocations not landing in `.root_hist` file and therefore there is no need to redefine it in custom `.rootrc`.

Reply:
Maybe because there is still the instance that was copied over before your remove it.  Either way adding or removing them is not related to the migration from Makefile to CMakeLists.  Please consider leaving them for now.

Reply:
( in addition, some `.rootrc` are more that just the setting of the root history files )

Reply:
Why? `ROOTTEST_ADD_TESTDIRS` is supposed to do it automatically.

Reply:
Moreover - `ROOTTEST_ADD_TESTDIRS` automatically create `.rootrc` for every sub-directory.
Therefore there is no need to have such generic `.rootrc` file in each test.